server/routes/index.ts

```diff
@@ -50,7 +50,7 @@ export function defineRoutes(router: IRouter, esClient: IClusterClient) {
   });
 
   const actionGroupSchema = schema.object({
-    description: schema.string(),
+    description: schema.maybe(schema.string()),
```

Comment:
This is not related to saml auth, right? Suggest move to a separate PR, and besides, please also remove `description` field for role/roleMapping/internalUser.

Reply:
Yes, let me remove it and post and other PR for that